### PR TITLE
Update sidecars to latest versions

### PR DESCRIPTION
**Pull request by ramineni**
**What this PR does / why we need it**:
This PR updates sidecar to latest versions.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by chrigl** [comment]
@ramineni Could you rebase? This should fix the failed job.

File: manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml [hunk 1]
```diff
@@ -47,7 +47,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.3.0
+          image: quay.io/k8scsi/csi-provisioner:v1.4.0
```
Comment (FengyunPan2) [inline manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml]: I remember that there are some bugs on csi-provisioner:v1.3.x.
We need 1.4 +, or latest, 1.4.0 maybe ok for us.

**Comment by FengyunPan2** [comment]
Merge it now.
/lgtm
/approve

**Pull request merged**